packages/components/src/components/input/input.stories.ts

```diff
@@ -69,731 +35,219 @@ export const Default = {
-          })}
-        </div>
-        <div class="mb-2">
-          ${generateTemplate({
-            constants: [
-              { type: 'attribute', name: 'type', value: 'url' },
-              { type: 'attribute', name: 'label', value: 'URL' },
-              { type: 'attribute', name: 'name', value: 'url field' },
-              { type: 'attribute', name: 'placeholder', value: 'https://www.union-investment.de/' },
-              { type: 'attribute', name: 'help-text', value: 'value must match url format' },
-              { type: 'attribute', name: 'form', value: 'testForm' },
-              { type: 'attribute', name: 'required', value: true },
-              {
-                type: 'slot',
-                name: 'right',
-                value: '<sd-icon slot="right" library="global-resources" name="system/picture"></sd-icon>'
-              }
-            ],
-            args
-          })}
-        </div>
-        <sd-button type="submit">Submit</sd-button>
-      </form>
-      <script>
-        function handleSubmit(event) {
-          const form = document.querySelector('#testForm');
-
-          const formData = new FormData(form);
-          const formValues = Object.fromEntries(formData);
-
-          if (form.reportValidity()) {
-            event.preventDefault(); // Prevent the default form submission behavior
-            alert('Form submitted with the following values: ' + JSON.stringify(formValues, null, 2));
-          }
-        }
-
-        document.querySelector('#testForm').addEventListener('submit', handleSubmit);
-      </script>
-    `;
-  }
+      <div class="mb-2">
+        <sd-input
+          type="search"
+          placeholder="^d{1,3}$"
+          label="Search"
+          help-text="use search format"
+          spellcheck
+        ></sd-input>
+      </div>
+      <div class="mb-2">
+        <sd-input
+          type="date"
+          placeholder="someone@example.com"
+          label="Date"
+          help-text="value is restricted to date format"
+          spellcheck
+        ></sd-input>
+      </div>
+      <div class="mb-2">
+        <sd-input
+          type="datetime-local"
+          placeholder="someone@example.com"
+          label="Date Time"
+          help-text="value is restricted to datetime format"
+          spellcheck
+        ></sd-input>
+      </div>
+      <div class="mb-2">
+        <sd-input type="time" label="Time" help-text="value is restricted to time format" spellcheck></sd-input>
+      </div>
+      <div class="mb-2">
+        <sd-input
+          type="number"
+          placeholder="^d{1,3}$"
+          label="Number"
+          help-text="value is restricted to numbers"
+          spellcheck
+        ></sd-input>
+      </div>
+      <div class="mb-2">
+        <sd-input
+          type="email"
+          placeholder="someone@example.com"
+          label="Email"
+          help-text="validate with email address format"
+          spellcheck
+        ></sd-input>
```

Comment:
Maybe we should change the validation text from german to english?
@mariohamann @coraliefeil 
![image](https://github.com/user-attachments/assets/3dc72b3b-2fb0-4c7e-b3bc-722a891d145a)

Reply:
@yoezlem from what I understood the language of the error message depends on the browser settings. For example, for me it is in english:
<img width="480" alt="image" src="https://github.com/user-attachments/assets/b6cd8274-e74f-4aeb-8504-81eadc09a681">